### PR TITLE
Add product bug and and remove flaky test from operator test

### PR DESCRIPTION
**Pull request by tarukumar**
Signed-off-by: Tarun Kumar <takumar@redhat.com>

### CODE REVIEW

**Comment by tarukumar** [comment]
Jenkins [job](https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1292) failed due to product bug

File: tests/Tests/100__deploy/130__operators/130__rhods_operator/130__rhods_operator.robot [hunk 1]
```diff
@@ -16,11 +16,12 @@ ${commercial_url}              https://www.redhat.com/en/technologies/cloud-comp
...
 Verify RHODS operator information
-  [Documentation]  This TC verfiy if the text present in RHODS opeartor Details section
-  [Tags]  ODS-498   ODS-624   Sanity    FlakyTest
+  [Documentation]  This TC verfiy if the text present in
+  ...   RHODS opeartor Details section.ProductBug:RHODS-4993
+  [Tags]  ODS-498   ODS-624   Sanity   ProductBug
   Open Installed Operators Page
   #Select All Projects
-  Click On Searched Operator      Red Hat OpenShift Data Science
+  Wait Until Keyword Succeeds    10 times  5s    Click On Searched Operator      Red Hat OpenShift Data Science   #robocp: disable
```
Comment (jgarciao) [inline tests/Tests/100__deploy/130__operators/130__rhods_operator/130__rhods_operator.robot:24]: typo in robocop

**Review by jgarciao — CHANGES_REQUESTED** [review]

File: tests/Tests/100__deploy/130__operators/130__rhods_operator/130__rhods_operator.robot (hunk 1 above)
Comment (tarukumar) [reply tests/Tests/100__deploy/130__operators/130__rhods_operator/130__rhods_operator.robot]: done

**Review by mwaykole — APPROVED** [review]

**Review by jgarciao — APPROVED** [review]
lgtm

**Pull request merged**